app/index.html

```diff
@@ -30,6 +34,11 @@
     <link
       rel="stylesheet"
       href="/static/style.css" />
+    <script>
+      function inDarkMode(){
+        return darkmode.inDarkMode;
+      }
+    </script>
```

Comment:
> does that load the browser's default settings?

It gets the current setting, whether it be browser default or user settings if overriden. It's all from [this](https://github.com/vinorodrigues/bootstrap-dark-5/blob/main/docs/darkmode.js.md#indarkmode) helper lib - I didn't know how to bind to that variable so this function gives me a way to bind. We shouldn't circumvent these libs since they manage the storage of user perefence - they should be our source of truth